test/sql/test_catalog/R/test_iceberg_hadoop_catalog

```diff
@@ -11,52 +11,63 @@ properties (
 "iceberg.catalog.warehouse"="oss://${oss_bucket}/test_catalog/test_iceberg_hadoop_catalog/${uuid0}"
 );
 -- result:
+[]
 -- !result
 create database ice_hadoop${uuid0}.ice_hadoop_db${uuid0};
 -- result:
+[]
 -- !result
 create table ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test (
     c0 int, 
     c1 string
 );
 -- result:
+[]
 -- !result
 insert into ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test values (1, "A");
 -- result:
+[]
 -- !result
 select * from ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test;
 -- result:
 1	A
 -- !result
 drop table ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test;
 -- result:
+[]
 -- !result
 create table ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test_double (
     c0 double
 );
 -- result:
+[]
 -- !result
 create table ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test_double_as as select * from ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test_double;
 -- result:
+[]
 -- !result
 describe ice_hadoop${uuid0}.ice_hadoop_db${uuid0}.test_double_as;
 -- result:
-c0	DOUBLE	Yes	false	None	
+c0	DOUBLE	Yes	false	None		None
```

Comment:
why result changed

Reply:
see this pr https://github.com/StarRocks/starrocks/pull/43360. it's merged